src/quo2/components/selectors/selectors.cljs

```diff
@@ -1,109 +1,73 @@
 (ns quo2.components.selectors.selectors
   (:require [quo2.components.icon :as icons]
-            [quo2.foundations.colors :as colors]
+            [quo2.components.selectors.styles :as style]
             [react-native.core :as rn]
-            [reagent.core :as reagent]
-            [quo2.components.selectors.styles :as style]))
+            [reagent.core :as reagent]))
 
 (defn- handle-press
-  [disabled? on-change checked?]
-  (when (not disabled?)
-    (fn []
-      (swap! checked? not)
-      (when on-change (on-change @checked?)))))
+  [on-change checked-atom checked?]
+  (when checked-atom (swap! checked-atom not))
+  (when on-change (on-change (not checked?))))
 
-(defn checkbox-prefill
-  [{:keys [default-checked?]}]
-  (let [internal-checked? (reagent/atom (or default-checked? false))]
-    (fn [{:keys [on-change disabled? blurred-background? container-style checked?]}]
```

Comment:
API changed:
In order to standarize our components, I renamed the `blurred-background?` property to `blur?` as we usually do